pkgs/development/python-modules/b2sdk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, setuptools_scm
+, requests, arrow, logfury, tqdm }:
+
+buildPythonPackage rec {
+  pname = "b2sdk";
+  version = "1.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0g527qdda105r5g9yjh4lxzlmz34m2bdz8dydqqy09igdsmiyi9j";
+  };
+
+  nativebuildInputs = [ setuptools_scm ];
+  propagatedBuildInputs = [ requests arrow logfury tqdm ];
+
+  # requires unpackaged dependencies like liccheck
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Client library and utilities for access to B2 Cloud Storage (backblaze).";
+    homepage = "https://github.com/Backblaze/b2-sdk-python";
+    license = licenses.mit;
```

Comment:
please add yourself as maintainer

Reply:
nope :)

Reply:
To clarify, I have no interest in manually maintaining random python libraries that should be autogenerated from lock files. If the pythonPackages maintainers think that is the best way to use maintainer time, they can do that of course.

Reply:
Lock files have their own negatives such as a closure size and large lock files which need to be committed, pretty much they are only good for developer time.

And python specifically has such low average release management quality per package they need to be manually patched to make them more usable.

thoughts @FRidh ?